README.md

```diff
@@ -1 +1,15 @@
-# DSCI_522_Group21
\ No newline at end of file
+# Fatality of Canadian Motor Vehicle Collisions Predictor
+
+## Project Proposal
+
+The data set that will be used in this project came from the National Collision Database published by Transport Canada and can be found [here](https://open.canada.ca/data/en/dataset/1eb9eba7-71d1-4b30-9fb1-30cbdab7e63a). The National Collision Database contains data on all of the police-reported motor vehicle collisions on public roads in Canada from 1999 to the most recent available data from 2017. This data set contains information licensed under the Open Government Licence – Canada.
+
+In this project we want to answer the research question, "Will a motor vehicle collision in Canada result in fatalities?"
```

Comment:
Can you add the type of the research question (and also sub-question(s)) as it is required in the instruction?

Reply:
I didn't include that because I wasn't sure what that meant (at least in the context of sub-questions). I can definitely add it for the research question though